src/user-guide/writing-workflows/external-triggers.rst

```diff
@@ -1,21 +1,47 @@
 .. _Section External Triggers:
 
-External Triggers (Xtriggers)
-=============================
+External Triggers
+=================
 
-Overview
---------
+Cylc :term:`dependencies <dependency>` allow us to trigger tasks off of
+events/outputs created within the workflow.
 
-Xtriggers allow tasks to trigger off of arbitrary external conditions. The
-:term:`scheduler` periodically calls a Python function, to check on the
-condition, until it returns success to satisfy dependent tasks.
+.. code-block:: cylc
 
+   [scheduling]
+       [[graph]]
+           # "bar" will wait until "foo" has succeeded
+           P1D = foo:succeeded => bar
 
-.. note::
-   Compared to the older :ref:`Old-Style External Triggers`, xtriggers are
-   visible in the graph, they are shared efficiently by all dependent tasks,
-   and they are entirely contained within the workflow.
+External triggers allow us to trigger tasks off of events that are external
+to the workflow itself, for example the real-world time (aka wallclock time),
+the arrival of a data set, or the completion of tasks in other workflows.
+
+.. code-block:: cylc
+
+   [scheduling]
+       [[graph]]
+           # "bar" will wait until the real-world time matches the cycle point
+           P1D = @wall_clock => bar
+
+Cylc provides two mechanisms for implementing external triggers in a workflow:
```

Comment:
Omitting the soft-deprecated (?) workflow-state triggers that we presumably don't want to advertise too prominently.